tracer/src/Datadog.Tracer.Native/cor_profiler.cpp

```diff
@@ -1381,16 +1381,6 @@ HRESULT STDMETHODCALLTYPE CorProfiler::Shutdown()
     return S_OK;
 }
 
-void CorProfiler::DisableTracerCLRProfiler()
```

Comment:
Are you sure this is safe to remove? Seems unrelated 🤔 (I would avoid removing it in case of old mismatch issues)

Reply:
Reverted, just in case.